backend/app/db/collectionWatcher.js

```diff
@@ -2,30 +2,30 @@ const MongoClient = require('mongodb').MongoClient;
 const EventEmitter = require('events');
 
 class CollectionWatcher extends EventEmitter {
-  constructor() {
-    super();
-    this.collections = [];
-  }
+	constructor() {
+		super();
+		this.collections = [];
+	}
 
-  async init(clientUrl) {
-    this.client = await MongoClient.connect(clientUrl);
-  }
-  subscribeToCollection(collection) {
-    const mongoCollection = this.client.db('rockandrolla').collection(collection);
-    const collectionStream = mongoCollection.watch();
+	async init(clientUrl) {
+		this.client = await MongoClient.connect(clientUrl);
+	}
+	subscribeToCollection(collection) {
+		const mongoCollection = this.client.db('tabley').collection(collection);
+		const collectionStream = mongoCollection.watch();
     
-    this.collections.push(
-      {
-       [collection]: collectionStream
-      });
+		this.collections.push(
+			{
+				[collection]: collectionStream
+			});
     
-    collectionStream.on('change', (change) => {
-      this.emit(`${collection}:change`, change);
-    });
-  }
+		collectionStream.on('change', (change) => {
+			this.emit(`${collection}:change`, change);
+		});
+	}
 
 }
 
 module.exports = {
-  CollectionWatcher
-}
\ No newline at end of file
+	CollectionWatcher
+};
\ No newline at end of file
```

Comment:
The code changes look good and do not have any obvious bugs or risks. Here are a few suggestions for improvement:

1. Add error handling - The current code does not handle any errors that might occur during the initiation of the MongoClient or while subscribing to a collection. It would be helpful to add appropriate error handling logic to catch any errors that might arise.

2. Use consistent indentation - The code uses a mixture of spaces and tabs for indentation, which may make it hard to read. It would be better to use one or the other consistently throughout the code.

3. Ensure correct database name - In the `subscribeToCollection` method, the MongoDB database name is hard-coded as 'tabley'. It is important to ensure that this is the correct name for your specific application.

Overall, the changes seem to improve readability and organization of the code.